src/averageAges.js

```diff
@@ -13,8 +13,26 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+function sum(years) {
+  return years.reduce((acumulator, person) => {
+    return acumulator + (person.died - person.born);
+  }, 0);
+}
+
 function calculateMenAverageAge(people, century) {
+  let manList = [];
+
+  century === undefined
```

Comment:
think of a way to remove undefined here
```suggestion
  century
```